GitUI/UserControls/FileStatusList.cs

```diff
@@ -780,6 +786,16 @@ protected override void DisposeCustomResources()
             _diffListSortSubscription?.Dispose();
         }
 
+        protected override void WndProc(ref Message m)
+        {
+            if (m.Msg == NativeConstants.WM_MOUSEACTIVATE)
+            {
+                _mouseEntered = !Focused;
```

Comment:
Changed this from simply `true` to `!Focused` as if you'd click on a list a second time this would get set to `true` and would remain so as `Enter` would not fire and this would not get reset to `False`.
Found this after some more manual testing.